extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -221,60 +240,59 @@ private function getParentWallMessageMock( array $params ) {
 			] )
 			->disableOriginalConstructor()
 			->getMock();
+		$parentWallMessageCounter = 0;
 
-		$parentWm->expects( $this->once() )
+
+		$parentWallMessageMock->expects( $this->at( $parentWallMessageCounter++ ) )
 			->method( 'load' );
-		$parentWm->expects( $this->once() )
+
+		$parentWallMessageMock->expects( $this->at( $parentWallMessageCounter++ ) )
 			->method( 'getTitle' )
-			->willReturn( $parentTitle );
-		$parentWm->expects( $this->any() )
+			->willReturn( $parentTitleMock );
+
+		$parentWallMessageMock->expects( $this->at( $parentWallMessageCounter++ ) )
 			->method( 'getMetaTitle' )
 			->willReturn( $params['parentWallMessage']['metaTitle'] );
-		$parentWm->expects( $this->once() )
+
+		$parentWallMessageMock->expects( $this->at( $parentWallMessageCounter++ ) )
+			->method( 'getUser' )
+			->willReturn( $parentUserMock );
+
+		$parentWallMessageMock->expects( $this->at( $parentWallMessageCounter++ ) )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this increment or decrement operator into a dedicated statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS881)